crispy-grid.gemspec

```diff
@@ -15,12 +15,11 @@ Gem::Specification.new do |s|
   # Gem Files
   s.files = %w(README.md MIT-LICENSE)
   s.files += Dir.glob("lib/**/*.*")
-  s.files += Dir.glob("stylesheets/**/*.*")
```

Comment:
By using Rails::Engine and the according directory structure, the stylesheets are imported in a Rails app.